### ISSUE TITLE
apperantly sql semicolon delimeter is kind of broken atm

### ISSUE THREAD

**Issue by Advencher** [zifter/clickhouse-migrations#15]
good time of the day,

so after 0.3.2 I had issues with same sql request files which worked fine with 0.3.0
for example this query 

```sql
create table testing.migration 
(
    `app_id`     Int16,
    `created_at` DateTime64(3, 'Europe/Moscow') CODEC (Delta(2), ZSTD(22)),
    `uid`        String CODEC (ZSTD(22)),
    `user_level` Nullable(Int64) CODEC (Delta(2), ZSTD(22)),
    `session_id` Nullable(String) CODEC (ZSTD(22)),
    uid_old      Nullable(String) CODEC (ZSTD(22)),
    id           Int64,
    level_old    Nullable(Int64),
    social       Nullable(String) CODEC (ZSTD(22))
)
    ENGINE = MergeTree()
        PARTITION BY toYYYYMM(created_at)
        PRIMARY KEY app_id
        ORDER BY (app_id, created_at)
        SETTINGS index_granularity = 8192;

create table testing.refund 
(
    `app_id`       Int16,
    `created_at`   DateTime64(3, 'Europe/Moscow') CODEC (Delta(2), ZSTD(22)),
    `uid`          String CODEC (ZSTD(22)),
    transaction_id Nullable(String) CODEC (ZSTD(22)),
    id             Int64
)
    ENGINE = MergeTree()
        PARTITION BY toYYYYMM(created_at)
        PRIMARY KEY app_id
        ORDER BY (app_id, created_at)
        SETTINGS index_granularity = 8192;
```

separates like this for some reason 

```sql
create table testing.migration         --that on is fine
(
    `app_id`     Int16,
    `created_at` DateTime64(3, 'Europe/Moscow') CODEC (Delta(2), ZSTD(22)),
    `uid`        String CODEC (ZSTD(22)),
    `user_level` Nullable(Int64) CODEC (Delta(2), ZSTD(22)),
    `session_id` Nullable(String) CODEC (ZSTD(22)),
    uid_old      Nullable(String) CODEC (ZSTD(22)),
    id           Int64,
    level_old    Nullable(Int64),
    social       Nullable(String) CODEC (ZSTD(22))
)
    ENGINE = MergeTree()
        PARTITION BY toYYYYMM(created_at)
        PRIMARY KEY app_id
        ORDER BY (app_id, created_at)
        SETTINGS index_granularity = 8192;
```
        
```sql
create table testing.refund 
(
    `app_id`       Int16,
    `created_at`   DateTime64(3, 'Europe/Moscow') CODEC (Delta(2), ZSTD(22)),
    `uid`          String CODEC (ZSTD(22)), 
    transaction_id Nullable(Strin   --this one is missing some if the lines|
 ```
 anyways, please ignore this if you know its on my end
 ty for reading

**Comment by zifter** [comment]
@Advencher I have added [9b40e4e](https://github.com/zifter/clickhouse-migrations/commit/9b40e4e27f238bc160356d620afc4c9dc0fd0ba1) you example to test - it looks like everything is ok. 

But I have some issues with clickhouse itself - this is line is pretty strange `user_level Nullable(Int64) CODEC (Delta(2), ZSTD(22)),`, because it causes error `Value of type Nullable(Int64) in memory is not of fixed size..`. So it's better to check.

Anyway, I suggest you to update to latest version 0.3.3 and if error still occurs send me logs, because I have added a bit more logs during execution migrations.
